app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -20,6 +20,11 @@ interface ReviewIssuesStepValue {
    */
   back: Blob | string | null | undefined;
 
+  /**
+   * Selfie image value.
+   */
+  selfie?: Blob | string | null | undefined;
```

Comment:
Probably related to my other comment,  but could you explain why selfie has a `?` here but the other two (`front` and `back`) don't, although all 3 have `?` in the interface defined in `document-capture-review-issues.tsx` above.

Reply:
The comment above is relevant too probably, but another part of the answer:
- The document-step can be submitted without front, back, or selfie.
- The review step, there must be a front and back. Selfie is optional.